src/routes/RemediationDetails.js

```diff
@@ -175,6 +181,41 @@ class RemediationDetails extends Component {
                                 </GridItem>
                             </Grid>
                         </StackItem>
+                        { localStorage.getItem('remediations:fifi:debug') === 'true' &&
```

Comment:
I don't think we need to guard this with a "feature flag". I think this change in mockups was intended to be a "lipstick" improvement, replacing the existing "System reboot" and "Playbook details" cards. The new card has nothing specific to FiFi in it so we can just release it.

Reply:
Cool. I'll remove the older bits in this PR too.